src/Button/ButtonGroup.tsx

```diff
@@ -1,10 +1,11 @@
 import styled from 'styled-components'
 import {get} from '../constants'
-import Box from '../Box'
+import Box, {BoxProps} from '../Box'
 import theme from '../theme'
 import sx from '../sx'
+import {ComponentProps} from '../utils/types'
 
-const ButtonGroup = styled(Box)`
+const ButtonGroup = styled(Box)<BoxProps>`
```

Comment:
```suggestion
const ButtonGroup = styled(Box)`
```

Reply:
Fixed in https://github.com/primer/components/pull/1017/commits/8996990bd01dffdd5058139da0c43a8ac76eb47d